src/data/reducers/ProfilePageReducer.js

```diff
@@ -1,21 +1,21 @@
 import {
-  SAVE_USER_PROFILE,
-  SAVE_USER_PROFILE_PHOTO,
-  DELETE_USER_PROFILE_PHOTO,
+  SAVE_PROFILE,
+  SAVE_PROFILE_PHOTO,
+  DELETE_PROFILE_PHOTO,
   EDITABLE_FIELD_CLOSE,
   EDITABLE_FIELD_OPEN,
+  FETCH_PROFILE,
 } from '../../actions/profile';
 
-
 const initialState = {
   error: null,
   saveState: null,
   savePhotoState: null,
   currentlyEditingField: null,
+  profile: {},
```

Comment:
Renamed this reducer to "profilePage" so that this part of its data is `profilePage.profile` instead of `profile.profile`.  Seems more accurate to what the reducer handles, too.